### ISSUE TITLE
Mid-year report - SARAdatascraping

### ISSUE THREAD

**Issue by gomezcatalina** [dfo-mar-odis/saraDataScraping#26]
### Proposed Change/Activity:

We need to provide updates on how things are going, as requested

### Actions:

- [x] Complete report [Catalina]
- [x] Review report [Quentin] 
- [x] Submit Report - November 4th - https://086gc.sharepoint.com/:w:/s/MaritimesSpatialPlanning-MAROpenDataLogistics/EdYo6fJNfv5CspruQnX2PpwB88BoESmt3z6u7yIhxrXD0Q?CID=6EE9EC1B-E4BF-429B-836A-6FE5501994F7&wdLOR=cB754EDB5-27C3-43E5-9D2C-50C599193226 [Catalina]

### Context

Outcomes of this project include:

1. Testing and evaluating the use of Reproducible Analytical Pipelines to automate data extraction from SARA recovery documents and Species at Risk geodatabase.  
2. Make recommendations regarding document elements (e.g. formatting or codes) required to identify and extract relevant information into a spreadsheet. This will coincide with the Species at Risk Program recovery and implementation team plans to revise Species’ Progress Report templates and other recovery document templates this fiscal year. 
3. Conceptual workflow of elements that would be required to reverse-engineer outcomes 1 and 2 (above) by using forms, csv files, or Excel spreadsheets to generate reproducible reports (e.g. using Microsoft PowerBI, R Markdown)

### Effort:
2

**Comment by stoyelq** [comment]
I went through the report and it looks good to me.  I had a couple minor thoughts: the instructions say we can adjust columns as necessary, but can we delete the ones with no values?  That would help a lot with the layout.  

Is it worth mentioning challenges around getting staff in Other?  This project went ahead because we could work on it, if we'd needed to hire someone as we proposed, it would not have started yet given how late in the timeline it got approved.
